store/migration/10001__init_schema.sql

```diff
@@ -1017,6 +1021,11 @@ UPDATE
     ON deployment_config FOR EACH ROW
 EXECUTE FUNCTION trigger_update_updated_ts();
 
+-- Define sheet enum types.
```

Comment:
It's intentional we don't use CREATE TYPE because they are not shared.

CREATE TYPE will create a separate object in Postgres, which is a overkill